draft-ietf-gnap-core-protocol.md

```diff
@@ -852,213 +847,76 @@ using the fictitious `photo-api` type definition.
                 "metadata",
                 "images"
             ]
-        }
-    ]
-~~~
-
-The access requested for a given object when using these fields 
-is the cross-product of all fields of the object. That is to 
-say, the object represents a request for all `action` values listed within the object
-to be used at all `locations` values listed within the object for all `datatype`
-values listed within the object. Assuming the request above was granted,
-the client instance could assume that it
-would be able to do a `read` action against the `images` on the first server
-as well as a `delete` action on the `metadata` of the second server, or any other
-combination of these fields, using the same access token. 
-
-To request a different combination of access, 
-such as requesting one `action` against one `location` 
-and a different `action` against a different `location`, the 
-client instance can include multiple separate objects in the `resources` array.
-The following non-normative example uses the same fictitious `photo-api`
-type definition to request a single access token with more specifically
-targeted access rights by using two discrete objects within the request.
-
-~~~
-    "resources": [
-        {
-            "type": "photo-api",
-            "actions": [
-                "read"
-            ],
-            "locations": [
-                "https://server.example.net/"
-            ],
-            "datatypes": [
-                "images"
-            ]
         },
         {
-            "type": "photo-api",
-            "actions": [
-                "write",
-                "delete"
-            ],
-            "locations": [
-                "https://resource.local/other"
-            ],
-            "datatypes": [
-                "metadata"
-            ]
-        }
-    ]
-~~~
-
-The access requested here is for `read` access to `images` on one server
-while simultaneously requesting `write` and `delete` access for `metadata` on a different
-server, but importantly without requesting `write` or `delete` access to `images` on the
-first server.
-
-It is anticipated that API designers will use a combination
-of common fields defined in this specification as well as
-fields specific to the API itself. The following non-normative 
-example shows the use of both common and API-specific fields as 
-part of two different fictitious API `type` values. The first
-access request includes the `actions`, `locations`, and `datatypes` 
-fields specified here as well as the API-specific `geolocation`
-field. The second access request includes the `actions` and
-`identifier` fields specified here as well as the API-specific
-`currency` field.
-
-~~~
-    "resources": [
-        {
-            "type": "photo-api",
+            "type": "walrus-access",
```

Comment:
What use is this `type`?

Reply:
The "type" field defines the API used. This structure is defined in section 8 now, but it's the same as it was before: https://deploy-preview-162--gnap-core-protocol-editors-draft.netlify.app/draft-ietf-gnap-core-protocol.html#name-resource-access-rights